tests/tacacs/test_accounting.py

```diff
@@ -57,16 +57,24 @@ def wait_for_log(host, log_file, pattern, timeout=20, check_interval=1):
     return []
 
 
-def check_tacacs_server_log_exist(ptfhost, tacacs_creds, command):
+def check_tacacs_server_log_exist(ptfhost, tacacs_creds, command, timeout=10):
     username = tacacs_creds['tacacs_rw_user']
     """
         Find logs run by tacacs_rw_user from tac_plus.acct:
             Find logs match following format: "tacacs_rw_user ... cmd=command"
             Print matched logs with /P command.
     """
     log_pattern = "/	{0}	.*	cmd=.*{1}/P".format(username, command)
-    logs = wait_for_log(ptfhost, "/var/log/tac_plus.acct", log_pattern)
-    pytest_assert(len(logs) > 0)
+
+    while timeout > 0:
+        logs = wait_for_log(ptfhost, "/var/log/tac_plus.acct", log_pattern)
```

Comment:
>wait_for_log [](http://example.com/codeflow?start=15&length=12)

wait_for_log already have default `timeout=20`. If this new `while` loop equal to `timeout=20*10` ? Is it too long? How does it help solve "TACACS UT randomly failed caused by timing issue" ?

Reply:
Fixed by revert this change.
Analyze the log and code again, seems another possible reason is: there are about 1.2 second delay between config change and hostcfgd render config file, so add code to wait hostcfgd update config file.